lib/conjur/conjur_config.rb

```diff
@@ -23,6 +23,7 @@ class ConjurConfig < Anyway::Config
     attr_config(
       # Read TRUSTED_PROXIES before default to maintain backwards compatibility
       trusted_proxies: (ENV['TRUSTED_PROXIES'] || []),
+      authorization_token_ttl: (ENV['CONJUR_AUTHORIZATION_TOKEN_TTL'] || []),
```

Comment:
This is only needed if we need environment variables to take precedence over the config file. As this is new functionality, we can handle it with the default behavior.  Also, we'll need to set the default time instead of an empty array.
```suggestion
      authorization_token_ttl: 480, # default is 8 minutes
```
```suggestion
      authorization_token_ttl: (ENV['CONJUR_AUTHORIZATION_TOKEN_TTL'] || []),
```

Reply:
done